providers/base/units/camera/jobs.pxu

```diff
@@ -119,13 +119,31 @@ estimated_duration: 20s
 depends: camera/detect
 requires: lsb.release >= '22.04'
 command:
-  timeout 120 camera_quality_test.py -d {{ name }} ||
-  (>&2 echo "Timeout computing score"; false)
+  camera_quality_test.py -d {{ name }}
```

Comment:
In cases where we want to save a file/document somewhere, we usually add an argument to specify where to save this. You could add an `-o/--output` argument for instance:

```suggestion
  camera_quality_test.py -d {{ name }} -o "${PLAINBOX_SESSION_SHARE}"
```